KsApi/models/UserEnvelope.swift

```diff
@@ -1,3 +1,16 @@
+import Foundation
+import ReactiveSwift
+
 public struct UserEnvelope<T: Decodable>: Decodable {
   public let me: T
 }
+
+// MARK: - GraphQL Adapters
+
+extension UserEnvelope {
```

Comment:
Transforming the Apollo clients `Data` type, for the given query, to a `SignalProducer` that emits our envelope.